hack/install_swagger.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This script is intended to be a convenience, to be called from the
+# Makefile `.install.swagger` target.  Any other usage is not recommended.
+
+BIN="$BINDIR/swagger"
+
+die() { echo "${1:-No error message given} (from $(basename $0))"; exit 1; }
+
+function install() {
```

Comment:
There is really no reason to make this a function. It's confusing to the reader.